cli/src/cmd/forge/script/build.rs

```diff
@@ -99,7 +99,7 @@ impl ScriptArgs {
 
         let mut extra_info = ExtraLinkingInfo {
             no_target_name,
-            target_fname,
+            target_fname: target_fname.clone(),
```

Comment:
This clone is for a better error message further down.. Once we clean up linking we should consider cleaning up the targetting logic in script as well (and other commands that take `<path>:<name>`)